client/capstone-app/src/components/Popup.jsx

```diff
@@ -23,21 +24,17 @@ const Popup = ({ isOpen, onOpenChange }) => {
     switch (action) {
       case "continue":
         onOpenChange(false);
-        console.log("Yes!");
         break;
       case "change-trip":
-        console.log("Change trip");
         navigate("/change-trip");
         break;
       case "delay-trip":
         setShowReportConfirmation(false);
         setShowBusSelection(true);
-        console.log("Bus late!");
         break;
       default:
         console.log("Unknown selection");
```

Comment:
I think Ryan G had this question in a past PR too--what's the plan to display this error to users? For instance, Facebook messaging has a little error message when messages fail to send. 

You don't need to implement it now, but it's user-friendly to consider how you'll surface errors to users. 

https://www.facebook.com/help/1024559617598844